tests/functional/test_lazy_loading.py

```diff
@@ -0,0 +1,99 @@
+import json
+import os
+import time
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+from datarobot_storage import get_storage
+from datarobot_storage.amazon import S3Storage
+from datarobot_storage.enums import FileStorageBackend
+
+from datarobot_drum.drum.lazy_loading.constants import BackendType
+from datarobot_drum.drum.lazy_loading.constants import EnumEncoder
+from datarobot_drum.drum.lazy_loading.constants import LazyLoadingEnvVars
+from datarobot_drum.drum.lazy_loading.lazy_loading_handler import LazyLoadingHandler
+from datarobot_drum.drum.lazy_loading.schema import LazyLoadingRepository
+from datarobot_drum.drum.lazy_loading.schema import S3Credentials
+
+
+class TestDownloadFromMinIO:
+    @pytest.fixture
+    def skip_minio_test_if_missing_env_vars(self):
+        if "AWS_ACCESS_KEY_ID" not in os.environ:
+            pytest.skip("AWS_ACCESS_KEY_ID not set in environment")
+        if "AWS_SECRET_ACCESS_KEY" not in os.environ:
+            pytest.skip("AWS_SECRET_ACCESS_KEY not set in environment")
+        if "MINIO_ENDPOINT_URL" not in os.environ:
+            pytest.skip("Minio endpoint URL not set in q")
+
+    @pytest.fixture
+    def repository(self, skip_minio_test_if_missing_env_vars):
```

Comment:
Do you need the `skip_minio_test_if_missing_env_vars` fixture here?

Reply:
Yes, because the given fixture tries to read expected environment variable: `os.environ[MINIO_ENDPOINT_URL]`

Reply:
But it looks like there's only 1 test in this module `test_download_from_minio_success` which already uses `skip_minio_test_if_missing_env_vars` fixture. So I'm saying that having it in `repository` fixture seems redundant